tests/test_upstream.c

```diff
@@ -116,6 +118,28 @@ ParodusCfg *get_parodus_cfg(void)
     return &parodusCfg;
 }
 
+char *getWebpaInterface(void)
+{
+    ParodusCfg cfg;
+    memset(&cfg,0,sizeof(cfg));
+    #ifdef WAN_FAILOVER_SUPPORTED
+        parStrncpy(cfg.webpa_interface_used , "wl0", sizeof(cfg.webpa_interface_used));
+    #else
+        parStrncpy(cfg.webpa_interface_used , "eth0", sizeof(cfg.webpa_interface_used));
+    #endif
+    set_parodus_cfg(&cfg);
+    #ifdef WAN_FAILOVER_SUPPORTED	
+	ParodusPrint("WAN_FAILOVER_SUPPORTED mode \n");
+	pthread_mutex_lock (&config_mut);	
+	parStrncpy(webpa_interface, get_parodus_cfg()->webpa_interface_used, sizeof(webpa_interface));
+	pthread_mutex_unlock (&config_mut);
+    #else
+	ParodusPrint("Erouter0 interface \n");
```

Comment:
Don't hardcode erouter0 in log as this is a generic code for pc and for all platforms , use interface variables if required